Makefile

```diff
@@ -0,0 +1,9 @@
+PROTO_ROOT := bridge/sdk-core/protos/api_upstream/
```

Comment:
Do you want both the Makefile and the Rakefile?

Reply:
Settled in favour of a Rakefile 👍